src/constants.js

```diff
@@ -25,7 +25,7 @@ const POOL_CONFIG_OPTIONS = Object.freeze([
   'fifo',
   'priorityRange',
   'autostart',
```

Comment:
Would be good to fix completely that configure option set instead of just fixing that one attribute nema.

Reply:
Also looks like these are only used in one place to emit some warnings... so I think it will be just fine to remove those warnings and let the tarn.js do the complaining if invalid attributes are passed to it.

Reply:
These options are already in `tarn.js`. Only changed one is `reapIntervalMillis`

```
min,
max,
acquireTimeoutMillis,
idleTimeoutMillis
```

Reply:
I've just removed `evictionRunIntervalMillis` from here. Since, this is only the source of warning. I think Its bad idea to remove warning.

```
this.mssqlPoolSettings = {
    min: 1,
    max: 1,
    idleTimeoutMillis: Number.MAX_SAFE_INTEGER,
    reapIntervalMillis: 0,
  };
```

Reply:
Currently that array is listing bunch of options that actually doesn't exist in tarn.js and allows to pass them and it fails to list other parateters that are supported by tarn. This kind warning of invalid parameters will be easier to maintain in tarn instead and it really shouldn't be any concern of knex.

It has been implemented becuase back in days knex swapped the pool implementation that it was using every other week.